### ISSUE TITLE
What is the alpha_matrix.csv?

### ISSUE THREAD

**Issue by nottwy** [farhat-lab/wdnn#7]
Recently I'm learning your tool and find that I can't understand what the file alpha_matrix.csv is.
It's used as target data in keras fit.
It's in line 44 of the file evaluation.py
`wdnn.fit(X_train, alpha_matrix[train], epochs=100, verbose=False, validation_data=[X_val,alpha_matrix[val]])`

**Comment by mchen16** [comment]
alpha_matrix is calculated based on the proportion of resistant isolates for each drug. In helpers.py, the function "masked_multi_weighted_bce()" takes in this alpha_matrix and reconstructs the labels used in the other machine learning models, so ultimately, the WDNN is trained on a similar label matrix as the other models. However, we adjusted our loss function to upweight the sparser class according to the proportion of resistant isolates in each drug, which is why we constructed the alpha_matrix in this way.
